react/src/components/ImportFromHuggingFaceModal.tsx

```diff
@@ -128,9 +128,12 @@ const ImportFromHuggingFaceModal: React.FC<ImportFromHuggingFaceModalProps> = ({
     queryKey: ['huggingFaceReadme', huggingFaceURL],
     queryFn: () => {
       if (_.isEmpty(huggingFaceURL)) return Promise.resolve({});
+      const searchParams = new URLSearchParams([
+        ['huggingface_url', huggingFaceURL ?? ''],
+      ]);
```

Comment:
If no reason to use array instead of object, please use object for readability.


```suggestion
      const searchParams = new URLSearchParams({
        huggingface_url: huggingFaceURL ?? '',
      });
```